src/main/resources/inspectionDescriptions/Codacy.html

```diff
@@ -1,28 +1,29 @@
-<!--TODO: technical writers-->
 <html>
 <body>
-Write your description here.
-Start the description with a verb in 3rd person singular, like reports, detects, highlights.
-In the first sentence, briefly explain what exactly the inspection helps you detect.
-Make sure the sentence is not very long and complicated.
+
 <p>
-    The first sentence must be in a dedicated paragraph separated from the rest of the text. This will make the
-    description easier to read.
-    Make sure the description doesn’t just repeat the inspection title.
+    This inspection is performed in the cloud by Codacy and detects and reports issues that affect the security, quality, and coverage of your code.
 </p>
+
 <p>
-    See https://jetbrains.design/intellij/text/inspections/#descriptions for more information.
+    Codacy is an automated code quality and coverage platform that analyzes your source code and identifies issues as you go, helping your team ship robust software by scanning over 40 programming languages, such as JavaScript, Python, Java, C#, and PHP.
 </p>
+
 <p>
-    Embed code snippets:
+    To use this inspection, make sure that the following prerequisites are met:
 </p>
-<pre><code>
-// automatically highlighted according to inspection registration 'language' attribute
-</code></pre>
-<!-- tooltip end -->
-<p>Text after this comment will only be shown in the settings of the inspection.</p>
 
-<p>To open related settings directly from the description, add a link with `settings://$` optionally followed by `?$` to
-    pre-select a UI element.</p>
+<ol>
+    <li>
+        You have a <a href="https://www.codacy.com/signup-codacy">Codacy account</a>.
```

Comment:
```suggestion
        You have a <a href="https://www.codacy.com">Codacy account</a>.
```

I'd rather go to the public root, not a specific page (not sure who chose `signup-codacy` but doesn't look like something that will endure the test of time)